### DISCUSSION TITLE
Frame generation. Is it possible?

### DISCUSSION THREAD

**Discussion by REB-ro** [Blinue/Magpie#852] [Ideas]
The lossless scaling application has a "frame generation" option, while doing everything else that Magpie already does (and in my opinion Magpie does better in terms of image quality).

The "frame generation" of lossless scaling seems to be a little different from the FG of FSR3 and DLSS3, I believe that the lossless scaling uses a technique similar to BFI (black frame insertion), but instead of inserting a black frame, it repeats a copy of the previous frame (or something like that), without interpolating. That's why it gives the feeling of greater fluidity in the game, like BFI does, but the frame pacing is a little strange.

While DLSS3 and FSR3 do interpolation for example Frame 1, Frame 1 + 2 (interpolated), Frame 2, Frame 2 + 3 (interpolated), Frame 3. The lossless scaling appears to do the following (not sure tho): Frame 1, Frame 1, Frame 2, Frame 2, Frame 3, Frame 3. But the feeling of fluidity does occur and pleases some people. And perhaps it would be interesting for Magpie to make this option available.

**Comment by hooke007** [comment]
https://github.com/Blinue/Magpie/issues/815
